crates/holochain_p2p/tests/tests/blocks.rs

```diff
@@ -325,6 +326,120 @@ async fn block_is_scoped_per_dna() {
         .unwrap());
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn get_from_blocked_agent_fails() {
+    let dna_hash = DnaHash::from_raw_32(vec![0xaa; 32]);
+    let keystore_1 = test_keystore();
+    let keystore_2 = test_keystore();
+    let TestCase { actor: actor_1, .. } = TestCase::new_with_keystore(&dna_hash, &keystore_1).await;
```

Comment:
What happened to our friends Alice and Bob??

Reply:
They were busy...

Reply:
They're available again https://github.com/holochain/holochain/pull/5328/commits/92ba1e1ab895c8f75f676af8bff5e0d37b5ec468